src/html/Input.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./input.css";
+
+// NOTE: Clearly not the full set yet ;)
+type InputType = "TEXT" | "NUMBER";
```

Comment:
given this is based on HTML attributes generally could to infer from the actual html attribute IMO

Reply:
Yes, definitely - however I wanted to constrain the options available, this is a heavily constrained `Input` component and would need extensive further work to make it reusable.

These types were simply to guide me in developing the date picker.